### PR TITLE
tetra: Implement getevents stdin mode

### PR DESCRIPTION
**Pull request by michi-covalent**
Import the code that was used to build the initial tetragon CLI, and
update it to reflect all the changes in the gRPC APIs. The initial CLI
has stopped working because the gRPC APIs have changed.

Ref: https://github.com/cilium/tetragon/releases/tag/tetragon-cli
Fixes: #438
Signed-off-by: Michi Mutsuzaki <michi@isovalent.com>

### CODE REVIEW

**Comment by michi-covalent** [comment]
you can test the CLI locally with:

```
make tetra
kubectl logs -n kube-system ds/tetragon -c export-stdout -f | ./tetra getevents -o compact
```

once this pr gets merge, i will:

- update README.md file.
- update the release process to include the CLI.
- work with the next release manager to get the CLI released 🚀

**Review by willfindlay — APPROVED** [review]

**Pull request merged**
